### DISCUSSION TITLE
Easy way of cleaning git history by removing very large files

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [community/community#20857] [Other Feature Feedback, Questions, & Ideas]
It is common behaviour to have messy github histories, and sometimes we upload large files that we then modify/remove, and this bloats the git history file. However, cleaning it as far as I know requires interacting with command line and seems a dangerous operation.

A feature on GitHub to easily remove large diffs or large files from the git-history would be super welcome, and will likely be more and more necessary as repos continue to grow in the years to come.

**Comment by radar-blip** [comment]
I agree that would be a welcome feature.
